MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -803,25 +803,11 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
                             break;
                         }
 
-                        ImageType imageType = artType switch
-                        {
-                            "banner" => ImageType.Banner,
-                            "clearlogo" => ImageType.Logo,
-                            "discart" => ImageType.Disc,
-                            "landscape" => ImageType.Thumb,
-                            "clearart" => ImageType.Art,
-                            // unknown type (including "poster") --> primary
-                            _ => ImageType.Primary,
-                        };
-
-                        Uri uri;
-                        try
-                        {
-                            uri = new Uri(val);
-                        }
-                        catch (UriFormatException ex)
+                        ImageType imageType = GetImageType(artType);
+
+                        if (!Uri.TryCreate(val, UriKind.Absolute, out var uri) || uri == null)
```

Comment:
```suggestion
                        if (!Uri.TryCreate(val, UriKind.Absolute, out var uri))
```
Can't be null when it returns true